shell/common/vsync_waiter.cc

```diff
@@ -109,6 +109,17 @@ void VsyncWaiter::FireCallback(fml::TimePoint frame_start_time,
     return;
   }
 
+  // The event named "VSYNC" is special in `chrome://tracing` tool - it will
```

Comment:
Why here?

This means the event won't get emitted on vsyncs where there's no callback, right?

I think making this more accurate is a good idea, but probably should fire unconditionally

Reply:
You are right! I have not thought about this problem before. Let me change it (probably within hours)